flask_dance/consumer/requests.py

```diff
@@ -144,6 +144,13 @@ def access_token(self):
         """
         return self.token and self.token.get("access_token")
 
+    @property
+    def refresh_token(self):
```

Comment:
Modifying the `OAuth2Session` class seems like a larger change than supporting this change from Dropbox. Can you move it to a separate pull request? It's also going to need an automated test and updated documentation to support it.

Reply:
this part of the PR is only thing not yet merged

rest was merged in some form in #339